hedera-mirror-rest/entityId.js

```diff
@@ -47,7 +47,8 @@ const maxEncodedId = 2n ** 63n - 1n;
 
 const entityIdRegex = /^(\d{1,5}\.){1,2}\d{1,10}$/;
 const encodedEntityIdRegex = /^\d{1,19}$/;
-const evmAddressRegex = /^(0x)?[A-Fa-f0-9]{40}$/;
+const evmAddressRegex = /^(\d{1,10}\.){0,2}[A-Fa-f0-9]{40}$/;
+const deprecatedEvmAddressInputRegex = /^(0x)?[A-Fa-f0-9]{40}$/;
```

Comment:
It's not deprecated, so suggest alternate naming.

```suggestion
const evmAddressRegex = /^(0x)?[A-Fa-f0-9]{40}$/;
const evmAddressShardRealmRegex = /^(\d{1,10}\.){0,2}[A-Fa-f0-9]{40}$/;
```

Reply:
renamed.